src/Catalyst.Core.Modules.Kvm/KvmModule.cs

```diff
@@ -45,8 +45,10 @@ protected override void Load(ContainerBuilder builder)
             
             builder.RegisterInstance(new MemDb()).As<IDb>(); // code db
             builder.RegisterInstance(new StateDb()).As<ISnapshotableDb>(); // state db
-            builder.RegisterType<CatalystGenesisSpec>().As<IReleaseSpec>();
-            builder.RegisterType<EthRpcService>().As<IEthRpcService>().SingleInstance();
+            
+            builder.RegisterType<StateReader>().As<IStateReader>(); // state db
+            
+            // builder.RegisterType<EthRpcService>().As<IEthRpcService>().SingleInstance();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4590253)